.github/workflows/e2e_on_pull.yaml

```diff
@@ -33,7 +33,9 @@ concurrency:
 jobs:
   authorize:
     runs-on: ubuntu-24.04
-    if: contains(github.event.pull_request.labels.*.name, 'test_e2e_libvirt')
+    if: |
+     contains(github.event.pull_request.labels.*.name, 'test_e2e_libvirt') ||
+     contains(github.event.pull_request.labels.*.name, 'test_e2e_docker')
```

Comment:
hi @stevenhorsman ! What if we have a 'test_e2e_all' too?

Reply:
I'm okay with this change as is. If in a sunny day, birds singing, my team won the last game then I can be motivated to improve it :)